etc/mpv-powershell-completion.ps1

```diff
@@ -0,0 +1,285 @@
+        }
+
+        $shortCommandName = $commandName.Substring(2)
+
+        $argName = ''
+
+        if ($commandName.EndsWith('='))
+        {
+            $shortCommandName = $shortCommandName.Substring(0, $shortCommandName.Length -1)
+        }
+        elseif ($commandName.Contains('='))
+        {
+            $shortCommandName = $shortCommandName.Substring(0, $shortCommandName.IndexOf('='))
+            $argName = $commandName.Substring($commandName.IndexOf('=') + 1)
+        }
+
+        foreach ($it in $UpdateOptions)
+        {
+            if ($shortCommandName -eq $it)
+            {
+                Update-Option $it
+                break
+            }
+        }
+
+        $results = New-Object Collections.Generic.List[Object]
+
+        $exactMatches = $Options | Where-Object { $_.name -eq $shortCommandName }
+
+        foreach ($it in $exactMatches)
+        {
+            if (-not $commandName.Contains('='))
+            {
+                continue
+            }
+
+            $arguments = $null
+
+            if ($it.type -eq 'flag')
+            {
+                $arguments = 'yes', 'no'
+            }
+
+            if ($it.type -eq 'choice' -and $null -ne $it.choices)
+            {
+                $arguments = $it.choices
+            }
+
+            if ($null -ne $arguments)
+            {
+                foreach ($arg in $arguments)
+                {
+                    if ($argName -ne '')
+                    {
+                        if ($arg.StartsWith($argName))
+                        {
+                            $results.Add('--' + $it.name + '=' + $arg)
+                        }
+                    }
+                    else
+                    {
+                        $results.Add('--' + $it.name + '=' + $arg)
+                    }
+                }
+            }
+        }
+
+        if (-not $commandName.Contains('='))
+        {
+            $partlyMatches = $Options | Where-Object { $_.name.StartsWith($shortCommandName) }
+
+            foreach ($it in $partlyMatches)
+            {
+                if ($it.name -eq $shortCommandName)
+                {
+                    continue
+                }
+
+                $results.Add('--' + $it.name)
+            }
+        }
+
+        return $results
+    }
+    elseif ($commandName -eq '')
+    {
+        return (Get-ChildItem).FullName | Resolve-Path -Relative
+    }
+    else
+    {
+        return (Get-ChildItem).FullName | Resolve-Path -Relative |
+            Where-Object { $_.ToLower().Contains($commandName.ToLower()) }
+    }
+}
+
+Register-ArgumentCompleter -Native -CommandName mpv -ScriptBlock {
+    param($commandName, $wordToComplete, $cursorPosition)
+
+    Get-Completion $cursorPosition "$wordToComplete" "$commandName" | ForEach-Object {
+        [System.Management.Automation.CompletionResult]::new($_, $_, 'ParameterValue', $_)
+    }
+}
+
+Register-ArgumentCompleter -Native -CommandName mpvnet -ScriptBlock {
```

Comment:
If I was the "native" mpv user, I would not like this.

Reply:
Why would you not like it, does not do harm, instead tries to help people.

Reply:
It's mpv's repo, right? I do not hope it would do irrelevant things.

Reply:
I agree, there shouldn't be any unrelated downstream code in mpv repo.